examples/server/rs-server.cpp

```diff
@@ -0,0 +1,134 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+
+#include <iostream>
+#include <csignal>
+#include <functional>
+
+#include <librealsense2/rs.hpp>
+#include <librealsense2-net/rs_net.hpp>
+
+#include "tclap/CmdLine.h"
+#include "tclap/ValueArg.h"
+
+#include <easylogging++.h>
+#ifdef BUILD_SHARED_LIBS
+// With static linkage, ELPP is initialized by librealsense, so doing it here will
+// create errors. When we're using the shared .so/.dll, the two are separate and we have
+// to initialize ours if we want to use the APIs!
+INITIALIZE_EASYLOGGINGPP
+#endif
+
+#define DBG CLOG(DEBUG,   "librealsense")
+#define ERR CLOG(ERROR,   "librealsense")
+#define WRN CLOG(WARNING, "librealsense")
+#define INF CLOG(INFO,    "librealsense")
+
+// Signal handler
+namespace {
+    std::function<void(int)> shutdown_handler;
+    void signal_handler(int signal) { shutdown_handler(signal); }
+}
+
+int main(int argc, char** argv)
+{
+    // Log engine initialization and configuration
+#ifdef BUILD_SHARED_LIBS
```

Comment:
#ifdefs shall not be used in demos, is it really necessary here?

Reply:
Moved to the tools.